client/src/pages/school/CreateReactTableForSchool.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { useTable, useSortBy } from "react-table";
+import { Table, Tbody, Th, Thead, Tr, Td, chakra } from "@chakra-ui/react";
+import processSchoolRow from "./processSchoolRow";
+import { TriangleDownIcon, TriangleUpIcon } from "@chakra-ui/icons";
+
+const CreateReactTableForSchool = ({ schoolData }) => {
+	// Prepare the data for React Table
+	const reactTableColumns = React.useMemo(
```

Comment:
Good to move the import of useMemo to line 1, so it would be `import React, {useMemo}` and then you can emit the React part in line 9. Looks a little cleaner.

Reply:
Done!

Reply:
This doesn't look to have been done when checking the code, potentially a git issue?

Same applies for my other comments